utils/temperusb/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2007-2018 OpenWrt.org
```

Comment:
no, this isn't true.  (unless you _really_ have copyright assigment with openwrt.org)  Use your own name, or simply drop it.

Reply:
ok.